client/cypress/cypress.js

```diff
@@ -33,13 +33,27 @@ function stopServer() {
 }
 
 function runCypressCI() {
-  const { PERCY_TOKEN_ENCODED, CIRCLE_REPOSITORY_URL } = process.env;
+  const {
+    PERCY_TOKEN_ENCODED,
+    CYPRESS_PROJECT_ID_ENCODED,
+    CYPRESS_RECORD_KEY_ENCODED,
+    CIRCLE_REPOSITORY_URL,
+  } = process.env;
 
-  if (PERCY_TOKEN_ENCODED && CIRCLE_REPOSITORY_URL && CIRCLE_REPOSITORY_URL.includes('getredash/redash')) {
-    process.env.PERCY_TOKEN = atob(`${PERCY_TOKEN_ENCODED}`);
+  if (CIRCLE_REPOSITORY_URL && CIRCLE_REPOSITORY_URL.includes('getredash/redash')) {
+    if (PERCY_TOKEN_ENCODED) {
+      process.env.PERCY_TOKEN = atob(`${PERCY_TOKEN_ENCODED}`);
+    }
+    if (CYPRESS_PROJECT_ID_ENCODED) {
+      process.env.CYPRESS_PROJECT_ID = atob(`${CYPRESS_PROJECT_ID_ENCODED}`);
+    }
+    if (CYPRESS_RECORD_KEY_ENCODED) {
+      process.env.CYPRESS_RECORD_KEY = atob(`${CYPRESS_RECORD_KEY_ENCODED}`);
+    }
   }
+
   execSync(
-    'docker-compose run cypress ./node_modules/.bin/percy exec -- ./node_modules/.bin/cypress run --browser chrome',
+    'docker-compose run cypress ./node_modules/.bin/percy exec -- ./node_modules/.bin/cypress run --record --browser electron',
```

Comment:
Electron is the default one, should we
```suggestion
    'docker-compose run cypress ./node_modules/.bin/percy exec -- ./node_modules/.bin/cypress run --record',
```
?

I see this is required for the recorded videos, previously I used chrome because one of the tests didn't run with Electron (it seems the issue is gone? if so, NP with this change)